src/main/java/net/neoforged/neoforge/items/ItemStackHandler.java

```diff
@@ -16,6 +20,10 @@
 public class ItemStackHandler implements IItemHandler, IItemHandlerModifiable, INBTSerializable<CompoundTag> {
     protected NonNullList<ItemStack> stacks;
 
+    public static Codec<ItemStackHandler> codec(HolderLookup.Provider provider) {
+        return new ItemStackHandlerCodec(provider);
+    }
```

Comment:
Again with only a codec and no stream codec? I can understand the reason why, but we shouldn't assume somebody will not need to synchronize themselves.

Reply:
You can wrap the existing (complicated) codec into a Stream codec if needed, but I can work on an implementation in a bit.